packages/contracts/foundry.toml

```diff
@@ -42,3 +42,4 @@ src = 'src/dollar'
 
 [profile.intense.fuzz]
 runs = 100000
+max_test_rejects = 90000
```

Comment:
@gitcoindev How did you come up with the `90000` number? By default it is [set](https://book.getfoundry.sh/reference/config/testing#max_test_rejects) to `65536`. Why do we need to increase it to this exact number?